/**:
  ros__parameters:
    # Total planning attempts
    max_solutions: 10

    # Planning group and link names
    arm_group_name: "panda_arm"
    eef_name: "hand"
    hand_group_name: "hand"
    hand_frame: "panda_link8"

    object1_name: "box"
    object2_name: "hammer"
    object1_mesh_path: "/home/ziyang/work_space/mesh_source/object1-copy.obj" 
    object2_mesh_path: "/home/ziyang/work_space/mesh_source/object2-copy.obj" 
    object3_mesh_path: "/home/ziyang/work_space/mesh_source/object1.obj" 

    # Poses
    hand_open_pose: "open"
    hand_close_pose: "close"
    arm_home_pose: "ready"

    # Scene frames
    world_frame: "world"
    table_reference_frame: "world"
    object_reference_frame: "world"
    surface_link: "table"

    # Collision object for picking
    # CYLINDER object specifications
    object_name: "object"
    object_dimensions: [0.25, 0.02] # [height, radius]
    object_pose: [0.5, -0.25, 0.0, 0.0, 0.0, 0.0]

    # Gripper grasp frame transform [x,y,z,r,p,y]
    grasp_frame_transform: [0.0, 0.0, 0.1, 3.141,0.0,0.785]

    # Place pose [x,y,z,r,p,y]
    place_pose: [0.6, -0.15, 0.0, 0.0, 0.0, 0.0]
    place_surface_offset: 0.0001 # place offset from table

    # Valid distance range when approaching an object for picking
    approach_object_min_dist: 0.1
    approach_object_max_dist: 0.4

    # Valid height range when lifting an object after pick
    lift_object_min_dist: 0.01
    lift_object_max_dist: 0.1
